nima/http2/webserver/src/test/java/io/helidon/nima/http2/webserver/ConnectionConfigTest.java

```diff
@@ -87,13 +69,12 @@ void testProviderConfigBuilder() {
 
         Http2ConnectionSelector provider = (Http2ConnectionSelector) Http2ConnectionProvider.builder()
                 .http2Config(DefaultHttp2Config.builder()
-                                     .maxFrameSize(4096L)
-                                     .maxHeaderListSize(2048L)
-                                     .build())
+                        .maxFrameSize(4096L)
```

Comment:
do not reformat existing code that is not modified please

Reply:
fixed